tasks/main.yml

```diff
@@ -22,15 +24,19 @@
     - volume
 
 - name: create filesystems
-  with_items: '{{ volume_partitions }}'
+  loop: '{{ volume_partitions }}'
+  loop_control: 
+    label: 'device: {{ item.device }}{% if item.format | default("") == "by-id" %}-part{% endif %}{{ item.number }}, fs: {{ item.filesystem }}'
   filesystem:
     fstype: '{{ item.filesystem }}'
     dev: '{{ item.device }}{% if item.format | default("") == "by-id" %}-part{% endif %}{{ item.number }}'
   tags:
     - volume
 
 - name: mount volumes
-  with_items: '{{ volume_partitions }}'
+  loop: '{{ volume_partitions }}'
+  loop_control:
+    label: 'device: {{ item.device }}{% if item.format | default("") == "by-id" %}-part{% endif %}{{ item.number }}, path: {{ item.path }}, options: {{ item.options | default([]) | join(",") }} '
```

Comment:
I think we can skip the options for the label

Reply:
removed